src/App/components/FormComponents/Checkbox.js

```diff
@@ -26,7 +26,7 @@ const Checkbox = ({ id, checked, name, disabled, groupTitle, label, errorMessage
                         <input {...attrs} id={id} />{"\n"}
                         <label htmlFor={id}>{label}</label>{"\n"}
                     </div>)}
-                    {errorMessage && <><div className="help-block">{errorMessage}</div>{"\n"}</>}
+                    {errorMessage && <><div className="help-block"><i className="material-icons">error</i>{errorMessage}</div>{"\n"}</>}
```

Comment:
According to figma, the icon should be a triangle 
![image](https://user-images.githubusercontent.com/44967773/174799459-13e2c673-b80b-45ec-abad-6375bf036054.png)